Extension/src/cppTools.ts

```diff
@@ -121,4 +121,8 @@ export class CppTools implements CppToolsTestApi {
     public getTestHook(): CppToolsTestHook {
         return test.getTestHook();
     }
+
+    public getClients() {
```

Comment:
This is not functionality that we intend to expose to extension authors at this time and we would provide no guarantees that we won't break this behavior again in the future.  We do not want extensions to tamper with our completion results because we do intend to add support for IntelliCode in a future release.